### PR TITLE
CONCF-615 | Breakpoints: AdminDashboard special pages list layout badly aligned

### PR DESCRIPTION
**Pull request by tomasznapieralski**

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
Please CR: @hakubo @rogatty

File: extensions/wikia/AdminDashboard/css/AdminDashboard.scss [hunk 1]
```diff
@@ -315,6 +314,9 @@ $color-highlight-column: rgba($color-page, 0);
 		font-style: normal;
 		margin: 0 30px 0 20px;
 	}
+    h2 {
```
Comment (rogatty) [inline extensions/wikia/AdminDashboard/css/AdminDashboard.scss]: Use `\t`

**Comment by rogatty** [comment]
@tomasznapieralski fix indentation and it LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by vforge** [comment]
LGTM

**Pull request merged**